src/libaktualizr/CMakeLists.txt

```diff
@@ -24,8 +24,8 @@ add_library(aktualizr_static_lib STATIC
     $<TARGET_OBJECTS:campaign>
     $<TARGET_OBJECTS:config>
     $<TARGET_OBJECTS:crypto>
-    $<TARGET_OBJECTS:http>
     $<TARGET_OBJECTS:jsoncpp>
+    $<TARGET_OBJECTS:http>
```

Comment:
Why break the alphabetical ordering?

Reply:
I will remove this, that's a relic of a previous version.